protos/sources/vega/commands/v1/commands.proto

```diff
@@ -132,6 +160,20 @@ message LiquidityProvisionAmendment {
   string reference = 6; // empty string means no change
 }
 
+// Amend a liquidity provision request
```

Comment:
Is this one unique to spot? If so, it should specify in the string.

Reply:
added